Recepies/Modules/Category/Details/View/Cells/FullDescriptionTableViewCell.swift

```diff
@@ -0,0 +1,95 @@
+// FullDescriptionTableViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Ячейка с подробным описанием рецепта
+final class FullDescriptionTableViewCell: UITableViewCell {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let veradanaFont = "Verdana"
+    }
+
+    static let reuseID = String(describing: FullDescriptionTableViewCell.self)
+
+    // MARK: - Visual Components
+
+    private let nameRecipeLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = .black
+        label.textAlignment = .left
+        label.font = UIFont(name: Constants.veradanaFont, size: 14)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        return label
+    }()
+
+    private let backGroundView: UIView = {
+        let view = UIView()
+        view.layer.cornerRadius = 25
+        view.clipsToBounds = true
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    private lazy var gradientLayer: CAGradientLayer = {
+        let gradientLayer = CAGradientLayer()
+        gradientLayer.colors = [UIColor.gradientBackground.cgColor, UIColor.white.cgColor]
+        return gradientLayer
+    }()
+
+    // MARK: - Initializers
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        configureView()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        configureView()
+    }
+
+    // MARK: - Life Cycle
+
+    override func layoutSublayers(of layer: CALayer) {
+        super.layoutSublayers(of: layer)
+        gradientLayer.frame = backGroundView.bounds
+    }
+
+    // MARK: - Public Methods
+
+    func setupCell(recipe: Recipe) {
+        nameRecipeLabel.text = recipe.description
+    }
+
+    // MARK: - Private Methods
+
+    private func configureView() {
+        backGroundView.layer.addSublayer(gradientLayer)
+        contentView.addSubview(backGroundView)
+        contentView.addSubview(nameRecipeLabel)
+        setNameRecipeLabelConstraint()
+        setBackGroundViewConstraint()
```

Comment:
```suggestion
        setNameRecipeLabelConstraint()
        setBackGroundViewConstraint()
```
В отдельный метод вынеси setConstraints()